tests/system/Config/fixtures/.env

```diff
@@ -6,15 +6,20 @@ NULL=
 
 SimpleConfig.onedeep=baz
 SimpleConfig.default.name=ci4
-SimpleConfig.simple.name=complex
+# Use underscore as separator
+SimpleConfig_simple_name=complex
+# Properity name with undersocre
+SimpleConfig.onedeep_value=bar
```

Comment:
I think another case is needed, eg

```
SimpleConfig.one_deep.under_deep
```

for user use "." but use underscored keys

Reply:
I added a test.